src/mcts/node.cc

```diff
@@ -526,8 +526,8 @@ V6TrainingData Node::GetV6TrainingData(
     GameResult game_result, const PositionHistory& history,
     FillEmptyHistory fill_empty_history,
     pblczero::NetworkFormat::InputFormat input_format, Eval best_eval,
-    Eval played_eval, Eval orig_eval, bool best_is_proven, Move best_move,
-    Move played_move) const {
+    Eval played_eval, bool best_is_proven, Move best_move, Move played_move,
```

Comment:
Just to make sure I don't miss things: To me it looks as if this overwrites `orig_eval` which is needed for value focus; or am I missing something crucial here?

Reply:
orig_eval was calculated outside this function, its now calculated inside this function.

Reply:
Thanks, this clarifies things.